kong/db/schema/entities/cas.lua

```diff
@@ -0,0 +1,41 @@
+local typedefs = require "kong.db.schema.typedefs"
+local openssl_x509 = require "openssl.x509"
+local ngx_time = ngx.time
```

Comment:
Note that we don't have to cache those global lookups in the Admin API server, it being a very cold path compared to our proxy server.